### ISSUE TITLE
Option to transfer the Wakaama repository to its own organisation

### ISSUE THREAD

**Issue by mlasch** [eclipse/wakaama#754]
As encouraged in this help desk ticket: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4225, we could move the `eclipse/wakaama` repository to its own `eclipse-wakaama/wakaama` organization.

As described in the ticket this would allow us to use the "otterdog self service": https://www.eclipse.org/projects/handbook/#resources-github-self-service. 

> This will capture the existing configuration as code in a separate repo (.eclipsefdn) and any committers can create PRs against this repo to change the configuration to fit your needs. The PRs will need to be approved by EF staff (and project leads if needed) before being merged and the changes finally being applied to GitHub. Here is for example a PR from another project enabling consistent branch protection rules for all repos: https://github.com/eclipse-velocitas/.eclipsefdn/pull/1 Here you can see a list of all Eclipse project that have this already enabled: https://eclipsefdn.github.io/otterdog-configs/ Please let us know if you'd like to proceed with switching to your own organisation and setting up self-service.

What do you think, should we go in this direction?

**Comment by sbernard31** [comment]
At Leshan have our own organization `eclipse-leshan` but we don't use **otterdog** for now.

We did that move to get access to github security advisories : https://github.com/eclipse-leshan/leshan/issues/1439

For us the move was very smoothly as described by eclipse IT : https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/449#note_1141465

HTH

**Comment by LukasWoodtli** [comment]
I think it's a good idea to have a distinct organization on GitHub.

The repo [wakaama-website](https://github.com/eclipse/wakaama-website) should also be moved into that organization.

btw: Does anyone know how

**Comment by sbernard31** [comment]
> The repo [wakaama-website](https://github.com/eclipse/wakaama-website) should also be moved into that organization.

Yep, this should be done by Eclipse Foundation. (I mean they do that by default when you ask for own organization)

> Does anyone know how that website is deployed?

This is manage by eclipse foundation.
This is  supposed to be a static website. If you change content of main branch then it will automatically be deployed few seconds?/minutes?/hours? later. If you observe that the site is not deployed 1 day later, you can notify eclipse foundation as you probably face a bug/issue.

**Comment by LukasWoodtli** [comment]
I would like to do that move to an own organization. Any thoughts about that?

**Comment by sbernard31** [comment]
I think there is good reason to do that. No veto on my side.

If you want to do it, just create an issue at : https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/new

**Comment by LukasWoodtli** [comment]
Opened an issue: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4499
